stream_test.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import generate_stream
+
+class GenerateStreamTest(unittest.TestCase):
+  def test_within_range_temp(self):
```

Comment:
These `within_range` tests check random number generation. They can pass even when the range is wrong.